_posts/2023-01-18-11.0.0-release.md

```diff
@@ -0,0 +1,137 @@
+* Improve performance by short-circuiting null checks when comparing non null field types ([#15106](https://github.com/apache/arrow/pull/15106))
+* Several enhancements to dictionary encoding ([#14891](https://github.com/apache/arrow/pull/14891), ([#14902](https://github.com/apache/arrow/pull/14902), ([#14874](https://github.com/apache/arrow/pull/14874))
+* Extend Table to support additional vector types ([#14573](https://github.com/apache/arrow/pull/14573))
+* Enhance and simplify handling of allocation management by integrating C Data into allocator hierarchy ([#14506](https://github.com/apache/arrow/pull/14506))  
+* Make ComplexCopier agnostic of specific implementation of MapWriter ([#14557](https://github.com/apache/arrow/pull/14557))
+* Distribute Apple M1 compatible JNI libraries via mavencentral ([#14472](https://github.com/apache/arrow/pull/14472))
+* Extend Table copy functionality, and support returning copies of individual vectors ([#14389](https://github.com/apache/arrow/pull/14389))
+
+## JavaScript notes
+
+* Bugfixes and dependency updates.
+* Arrow now requires BigInt support. [GH-33681](https://github.com/apache/arrow/pull/33682)
+
+## Python notes
+
+New features and improvements:
+
+* Numpy conversion for ListArray is improved taking into account sliced offset [(GH-20512)](https://github.com/apache/arrow/issues/20512)
+* DataFrame Interchange Protocol is implemented for ``pyarrow.Table`` ([GH-33346](https://github.com/apache/arrow/issues/33346)).
+
+## R notes
+* map_batches() is lazy by default; it now returns a RecordBatchReader instead of a list of RecordBatch objects unless lazy = FALSE. [GH-14521](https://github.com/apache/arrow/issues/14521)
+* A substantial reorganisation, rewrite of and addition to, many of the vignettes and README. [GH-14514](https://github.com/apache/arrow/issues/14514)
+
+For more on what’s in the 11.0.0 R package, see the [R changelog][4].
+
+## Ruby and C GLib notes
+
+### Ruby
+
+* `Arrow::Table#save` now always returns self instead of the result of its `raw_records`[GH-15289](https://github.com/apache/arrow/issues/15289)
+* Improve the GC-related crash prevention system by guarding the shared objects from GC [ARROW-18161](https://issues.apache.org/jira/browse/ARROW-18161)
+* Add `Arrow::HalfFloat` and `raw_records` support in `Arrow::HalfFloatArray` [ARROW-18086](https://issues.apache.org/jira/browse/ARROW-18086)
+* Support omitting join keys in `Table#join` [GH-15084](https://github.com/apache/arrow/issues/15084)
+* Add support for `Arrow::Table.load(uri, schema:)` [ARROW-15206](https://issues.apache.org/jira/browse/ARROW-15206)
+* Add `Arrow::ColumnContainable#column_names` (e.g. `Arrow::Table#column_names`) [GH-15085](https://github.com/apache/arrow/issues/15085)
+* Add `to_arrow_chunked_array` methods to support converting to `Arrow::ChunkedArray` [ARROW-18405](https://issues.apache.org/jira/browse/ARROW-18405)
+
+### C GLib
```

Comment:
```suggestion
### C GLib

* Add `garrow_chunked_array_new_empty()` [GH-33671](https://github.com/apache/arrow/issues/33671)
* Add `GArrowProjectNodeOptions` [GH-33670](https://github.com/apache/arrow/issues/33670)
* Add `GADatasetHivePartitioning` [GH-15257](https://github.com/apache/arrow/issues/15257)
* The signature of `garrow_execute_plain_wait()` was changed to take the `error` argument and to return the finished status [GH-15254](https://github.com/apache/arrow/issues/15254)
* Add support for half float [GH-15168](https://github.com/apache/arrow/issues/15168)
* Add `GADatasetFinishOptions` [GH-15146](https://github.com/apache/arrow/issues/15146)
```

Reply:
@kou Could you check the summary of C GLib changes?

Reply:
Thanks!